yaeger.spacesimulator/src/main/resources/config.properties

```diff
@@ -8,4 +8,6 @@ base-planet-volume = 500
 base-planet-density = 25
 base-planet-color = 0xF1F1F1
 
-gravity-constant = 0.2D
\ No newline at end of file
+gravity-constant = 0.2D
+
+ui-vieworder = 1
```

Comment:
Bedoel je hiermee iets als z-index? Dat kan je ook realiseren door addEntity in een bepaalde volgorde aan te roepen, 1e addEntity gaat meest naar achter, 2e over 1e, 3e over 2e, etc..

Reply:
ah nvm je plaatst altijd de pauzebutton VOOR simulatieobjecten idd